sw/device/lib/dif/dif_spi_device.h

```diff
@@ -22,18 +22,32 @@ extern "C" {
 #endif  // __cplusplus
 
 /**
- * A signal edge type: positive or negative.
+ * Clock polarity type
```

Comment:
Please make sure the comments are consistent with others in this file. I know they're verbose, but consistency is important. E.g., `A clock polarity type: positive or negative.`